.github/workflows/master-push.yml

```diff
@@ -53,3 +53,17 @@ jobs:
             k=$(nix build .#k --json | jq -r '.[].outputs | to_entries[].value')
             drv=$(nix-store --query --deriver ${k})
             nix-store --query --requisites --include-outputs ${drv} | cachix push k-framework
+
+      - name: 'Update dependents'
+        env:
+          GITHUB_TOKEN: ${{ secrets.JENKINS_GITHUB_PAT }}
+        run: |
+          set -x
+          version="${GITHUB_SHA}"
+          curl --fail                                                          \
+            -X POST                                                            \
+            -H "Accept: application/vnd.github+json"                           \
```

Comment:
This should run in the `release.yml` workflow, and replace this: https://github.com/runtimeverification/k/blob/master/.github/workflows/release.yml#L345

Note there that we also are not using `GITHUB_SHA` for the version, but the output from a previous stage.

Reply:
This should be removed now.